main/mafia.hs

```diff
@@ -500,14 +502,39 @@ ghciArgs extraIncludes paths = do
 
   extras <- concat <$> mapM reifyInclude extraIncludes
 
-  let dirs = ["src", "test", "gen", "dist/build/autogen"] <> extras
-  includes  <- catMaybes <$> mapM ensureDirectory dirs
+  let
+    dirs =
+      ["src", "test", "gen", "dist/build/autogen"] <> extras
```

Comment:
I *think* you still need the extra one for `dist/build/autogen` (load path is dumb) but will check